src/browser/port_discovery.ts

```diff
@@ -36,7 +37,7 @@ export class PortDiscovery extends EventEmitter {
                 log.writeToLog('info', 'Constructing the copyDataTransport window.');
                 this._transport = new WMCopyData(WINDOW_CLASS_NAME, WINDOW_CLASS_NAME);
             } else {
-                // TODO: provide a unix implementation
+                this._transport = new UnixDomainSocket(UNIX_FILE_DESCRIPTOR_PREFIX);
```

Comment:
To provide some clarification on how this works: here the core is going to create the unix domain socket transport, which means it'll create a server that is bound to `UNIX_FILE_DESCRIPTOR_PREFIX` + `Date.now()`. Other runtimes will be able to find this server by querying for all open file descriptors with this prefix.